portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSet.path

```diff
@@ -113,6 +118,11 @@
 	<td>//div[@class='dropdown-menu show']//*[@type="checkbox" and @aria-label='${key_status}']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>RESET_FILTERS</td>
+	<td>//*[@class='tbar-item']//button[@class='btn btn-unstyled']</td>
+	<td></td>
```

Comment:
Can we have the path be like `//*[contains(@class,'management-bar')]//*[@type='button' and contains(text(), 'Reset Filters')]` instead? :)

Reply:
OK, I will update the path and re-pull the commit.